Makefile

```diff
@@ -27,13 +27,15 @@ README.md: index.js
 .PHONY: lint
 lint:
 	$(ESLINT) \
+	  --env es6 \
```

Comment:
I just noticed that this is `es6` rather than `es3`. Let's keep it as `es3` to ensure that we don't accidentally sneak ES6 features into the source code (it's okay to use ES6 features in the test suite).

Please update the commit with `git commit --amend`. You'll then need to use `--force` when you push to your branch.